pkg/resources/stream_on_table_acceptance_test.go

```diff
@@ -0,0 +1,564 @@
+						HasTableName(table.ID().FullyQualifiedName()).
+						HasSourceType(sdk.StreamSourceTypeTable).
+						HasBaseTables([]sdk.SchemaObjectIdentifier{table.ID()}).
+						HasType("DELTA").
+						HasStale("false").
+						HasMode(sdk.StreamModeDefault).
+						HasStaleAfterNotEmpty().
+						HasInvalidReason("N/A").
+						HasOwnerRoleType("ROLE"),
+					assert.Check(resource.TestCheckResourceAttrSet(resourceName, "describe_output.0.created_on")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.name", id.Name())),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.database_name", id.DatabaseName())),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.schema_name", id.SchemaName())),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.owner", snowflakeroles.Accountadmin.Name())),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.comment", "")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.table_name", table.ID().FullyQualifiedName())),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.source_type", "Table")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.base_tables.#", "1")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.base_tables.0", table.ID().FullyQualifiedName())),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.type", "DELTA")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.stale", "false")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.mode", string(sdk.StreamModeDefault))),
+					assert.Check(resource.TestCheckResourceAttrSet(resourceName, "describe_output.0.stale_after")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "describe_output.0.owner_role_type", "ROLE")),
+				),
+			},
+			// import without optionals
+			{
+				Config:       config.FromModel(t, model),
+				ResourceName: resourceName,
+				ImportState:  true,
+				ImportStateCheck: assert.AssertThatImport(t,
+					resourceassert.ImportedStreamOnTableResource(t, resourceId).
+						HasNameString(id.Name()).
+						HasDatabaseString(id.DatabaseName()).
+						HasSchemaString(id.SchemaName()).
+						HasFullyQualifiedNameString(id.FullyQualifiedName()).
+						HasAppendOnlyString(r.BooleanFalse).
+						HasTableString(table.ID().FullyQualifiedName()),
+				),
+			},
+			// set all fields
+			{
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_StreamOnTable/at"),
+				ConfigVariables: tfconfig.ConfigVariablesFromModel(t, model.WithCopyGrants(true).WithComment("foo").WithAppendOnly(r.BooleanTrue).WithShowInitialRows(r.BooleanTrue).WithAtValue(pluginconfig.MapVariable(map[string]pluginconfig.Variable{
```

Comment:
hmm, that's my opinion, but I think it would be better to have all the models at the top. Yes, you have to scroll, but it's easier to read and compare them when they're next to each other.

Reply:
Yeah, as we discussed today, I'll move this to top.